app/controllers/sessions_controller.rb

```diff
@@ -13,13 +13,13 @@ def create
         redirect_to dashboard_path(id: @user.id)
       end
     else
-      flash.now[:danger] = "Invalid. Try Again."
+      flash.now[:danger] = "Invalid login details. Please try again."
       render :new
     end
   end
 
   def destroy
-    session.clear
+    session[:user_id] = nil
```

Comment:
Can you explain this line to me later? Is this more helpful/clear in some way?
session.clear seems more explicit. I'm assuming it clears out something we don't want to lose.